packages/framer-motion/src/events/use-dom-event.ts

```diff
@@ -4,7 +4,7 @@ export function addDomEvent(
     target: EventTarget,
     eventName: string,
     handler: EventListener,
-    options?: AddEventListenerOptions
+    options: AddEventListenerOptions = { passive: true }
```

Comment:
This is not new, but I'm surprised the interface matches `removeEventListener`  (line:11), I expect add & removeEventListener to have different option types 🤔

Reply:
Good point, I think they do, not sure why that wasn't flagging